hybridse/src/udf/udf.cc

```diff
@@ -184,6 +184,51 @@ int32_t weekofyear(Date *date) {
     }
 }
 
+void last_day(int64_t ts, Date *output, bool *is_null) {
+    time_t time = (ts + TZ_OFFSET) / 1000;
+    struct tm t;
+    if (nullptr == gmtime_r(&time, &t)) {
+        *is_null = true;
+        return;
+    }
+    gmtime_r(&time, &t);
+    try {
+        boost::gregorian::date d = boost::gregorian::date_from_tm(t);
```

Comment:
Since there is previously a Seg fault issue in https://github.com/4paradigm/OpenMLDB/pull/2159#issuecomment-1185081618 that was sadly pointed to the boost datetime, so I recommend implement it with abseil time instead.  Similar as I previously did in https://github.com/4paradigm/OpenMLDB/pull/2175. We hope boost can be removed in the future.

Tell me If you have any other thoughts :)

Reply:
Thanks for your detailed instruction!